styles.css

```
@@ -0,0 +1,81 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap");
+
+.nav-bar {
+  display: flex;
+  max-width: 375px;
+  justify-content: space-between;
+  background-color: #3c3a39;
+  text-decoration: solid;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+  color: #fff;
+  text-decoration: solid;
+  font-weight: 700;
+  line-height: 24px;
+  font-family: "Roboto", sans-serif;
+  font-size: 18px;
+  align-items: center;
+  margin-left: 10px;
+}
+
+.menu {
+  margin-right: 10px;
+}
+
+h1 {
+  padding-top: 30px;
+  color: #ff6b00;
+  width: 300px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 30px;
+  line-height: 50px;
+  margin-bottom: 0;
+}
+
+h3 {
+  width: 300px;
+  height: 24px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 10px;
+  color: #fff5e1;
+  margin-bottom: 0;
+}
+
+p {
+  width: 300px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 22px;
+  color: #fff;
+}
+
+.content {
+  display: flex;
+  max-width: 375px;
+  flex-direction: column;
+  align-items: center;
+  background: #1c1a19;
+  padding-bottom: 100px;
+  background-image: url("illustration.png");
```

- [ ] Here's the typo 🤓 

```suggestion
  background-image: url("./Illustration.png");
```